### PR TITLE
Issue-956 Fetch registry from graph and refactor balance formatting

### PR DESCRIPTION
**Pull request by ishrak-imam**
### Description

Fetch registry information from Litentry graph and refactor the balance formatting util functions accordingly.

### CODE REVIEW

File: src/api/utils/balance.ts [hunk 1]
```diff
@@ -1,21 +1,18 @@
...
- * turns a string to a Balance BN in order to send through API or
- * show the formatted version to user
- * @param api
- * @param input String (user input E.g. 2.33 or 20 or 0.33)
- * @returns BN
+ * use this method to get BN from local input strings
  */
-export function getBalanceFromString(api: ApiPromise, input: string): BN {
-  const chainDecimal = api.registry.chainDecimals[0] ?? 10;
+export function getBNFromLocalInputString(registry: Registry, input: string): BN {
```
Comment (silva-fj) [inline src/api/utils/balance.ts]: How about `stringToBn` ?

File: src/api/utils/balance.ts [hunk 2]
```diff
@@ -1,21 +1,18 @@
...
+import type {Registry} from 'src/api/hooks/useChainInfo';
+import {bnToBn} from '@polkadot/util';
 
 /**
- * turns a string to a Balance BN in order to send through API or
- * show the formatted version to user
- * @param api
- * @param input String (user input E.g. 2.33 or 20 or 0.33)
- * @returns BN
+ * use this method to get BN from local input strings
```
Comment (silva-fj) [inline src/api/utils/balance.ts]:
```suggestion
 * Use this method to convert unformatted string values (Balance) to BN. 
 *  e.g: amount entered in a TextInput
```

File: src/api/utils/balance.ts [hunk 3]
```diff
@@ -25,3 +22,10 @@ export function getBalanceFromString(api: ApiPromise, input: string): BN {
 
   return new BN(input.replace(/[^\d]/g, '')).mul(BN_TEN.pow(currencyPower));
 }
+
+/**
+ * use this method to get BN from string values incoming from API
+ */
+export function getBNFromApiString(value?: string) {
```
Comment (silva-fj) [inline src/api/utils/balance.ts]: How about `formattedStringToBn`?

File: src/api/utils/balance.ts [hunk 4]
```diff
@@ -25,3 +22,10 @@ export function getBalanceFromString(api: ApiPromise, input: string): BN {
 
   return new BN(input.replace(/[^\d]/g, '')).mul(BN_TEN.pow(currencyPower));
 }
+
+/**
+ * use this method to get BN from string values incoming from API
```
Comment (silva-fj) [inline src/api/utils/balance.ts]:
```suggestion
 * Use this method to convert formatted string values (Balance) to BN. 
 * e.g: values coming from the litentry-graph
```

File: src/hooks/useFirebase.tsx [hunk 5]
```diff
@@ -26,7 +26,7 @@ export function useFirebase() {
     throw new Error('InAppNotificationContext most be provided!');
   }
 
-  const handleNotifiction = useCallback(
```
Comment (silva-fj) [inline src/hooks/useFirebase.tsx:29]: 😆

Comment (ishrak-imam) [reply src/hooks/useFirebase.tsx:29]: 🤣

File: src/ui/screens/DemocracyScreen.tsx [hunk 6]
```diff
@@ -21,8 +21,8 @@ export function DemocracyScreen() {
 
   const groupedData: {title: string; data: Array<DemocracyProposal | DemocracyReferendum>}[] = React.useMemo(
     () => [
-      {title: 'Referenda', data: democracy.proposals ?? []},
-      {title: 'Proposals', data: democracy.referendums ?? []},
+      {title: 'Referenda', data: democracy.referendums ?? []},
+      {title: 'Proposals', data: democracy.proposals ?? []},
```
Comment (ishrak-imam) [inline src/ui/screens/DemocracyScreen.tsx:25]: Sneaky bug fix. Including this here because very tiny fix.

**Review by silva-fj — APPROVED** [review]

**Pull request merged**